src/forwarder.rs

```diff
@@ -153,25 +157,54 @@ fn recv_from_channel_and_send_multiple_dest(
         |_received_packet, _is_already_marked_as_discard, _is_dup| {},
     );
 
+    let (trace_shreds, packets): (Vec<TraceShred>, Vec<Packet>) = packet_batch_vec[0]
```

Comment:
if we just got the TraceShred packets here, we might be able to avoid a cloned?

Reply:
if you make this Vec<&Packet> maybe could avoid the cloned?

Reply:
discussed offline, will fwd all pkts to validator